aicsimageio/tests/readers/test_ome_tiff_reader.py

```diff
@@ -50,6 +53,19 @@
             [f"C:{i}" for i in range(10)],  # This is the actual metadata
             (1.0, 1.0, 1.0),
         ),
+        (
+            # This is actually an OME-TIFF file
+            # Shows we don't just work off of extensions
+            # But the content of the file
+            "s_1_t_1_c_2_z_1_RGB.tiff",
+            "Image:0",
+            ("Image:0",),
+            (1, 2, 1, 32, 32, 3),
+            np.uint8,
+            dimensions.DEFAULT_DIMENSIONS_ORDER_WITH_SAMPLES,
+            [None, None],
+            (1.0, 1.0, 1.0),
+        ),
```

Comment:
Thanks `AICSImage` object for showing me that this file has an incorrect extension :upside_down_face: